modules/luci-mod-status/htdocs/luci-static/resources/view/status/channel_analysis.js

```diff
@@ -314,6 +314,22 @@ return view.extend({
 					if (res.vht_operation.channel_width == 80) {
 						chan_width = 8;
 						res.channel_width = "80 MHz";
+
+						/* If needed, adjust based on the newer interop workaround. */
+						if (res.vht_operation.center_freq_2) {
+							var diff = res.vht_operation.center_freq_2 -
+								res.vht_operation.center_freq_1;
```

Comment:
This uses inconsistent alignment: it should not be all tabs, but only tabs until the same column as the start of "var" on line 320 and spaces from then on.

(tabs for indentation and spaces for spaces for alignment)

Reply:
Fixed :)